flink-runtime/src/test/java/org/apache/flink/runtime/persistence/TestingStateHandleStore.java

```diff
@@ -121,11 +121,6 @@ public boolean releaseAndTryRemove(String name) throws Exception {
         return removeFunction.apply(name);
     }
 
-    @Override
-    public void releaseAndTryRemoveAll() throws Exception {
-        removeAllRunnable.run();
```

Comment:
`removeAllRunnable` can be removed entirely

Reply:
Done.

Reply:
@XComp Thanks for your review.